test/general/testResults.ts

```diff
@@ -555,10 +553,10 @@ mParticle.logEvent('CustomSocial', mParticle.EventType.Social)
 mParticle.logEvent('CustomOther', mParticle.EventType.Other)
 
 // Data Plan Point 10
-mParticle.logEvent('CustomUnknown', mParticle.EventType.Media)
+mParticle.logEvent('CustomUnknown', mParticle.EventType.Unknown)
 
 // Data Plan Point 11
-mParticle.logEvent('CustomMedia', mParticle.EventType.Other)
+mParticle.logEvent('CustomMedia', mParticle.EventType.Media)
```

Comment:
I think there should still be a test for `Other` since it is a valid event type

Reply:
I believe `Data Plan Point 9` tests for `Other` for each platform.

Reply:
ah, got it. didn't see that because it was hidden.  thanks for pointing that out!